database/users_models.py

```diff
@@ -57,7 +57,7 @@ class Permission(db.Model):
 # Define a user table to store the user's data when they sign up for the saving challenges algorithm
 class UserInformation(db.Model):
   __tablename__ = 'user_information'
```

Comment:
Please format this with `black` so that this is double quoted

Reply:
The black Python formatter https://black.readthedocs.io/en/stable/getting_started.html

Activate the virtual environment, then do `pip install black`. Then run `black path-to-file/users_model.py` to format this file. You can also setup the formatting in PyCharm, just google how.